src/posts/2012-02-28-updates-to-ildh.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: layouts/post
+title: Updates to the Inclusive Learning Design Handbook
+date: '2012-02-28'
+tags: post
```

Comment:
What are tags and how are they being used in this instance of 11ty? Is the `post` tag being used for a function?

Reply:
They are used to make a collection which is then being used to render all the news articles on the news index page.The code which is using the tags is : 
    {%- for post in collections.post reversed -%}
    <li><a href="{{ '/' | url }}{{ post.data.permalink }}"><p> {{ post.data.title }}</p></a>
    <time class="floe-date" datetime="{{ post.data.date | w3DateFilter }}">{{ post.data.date | dateFilter }}</time>
    </li>
    {%- endfor -%}